library/alloc/src/vec.rs

```diff
@@ -2566,7 +2566,7 @@ __impl_slice_eq1! { [const N: usize] Vec<A>, &[B; N], #[stable(feature = "rust1"
 //__impl_slice_eq1! { [const N: usize] Cow<'a, [A]>, &[B; N], }
 //__impl_slice_eq1! { [const N: usize] Cow<'a, [A]>, &mut [B; N], }
 
-/// Implements comparison of vectors, lexicographically.
+/// Implements comparison of vectors, [lexicographically](core::cmp::std/cmp/trait.Ord.html#lexicographical-comparison).
```

Comment:
This is a broken link.
```suggestion
/// Implements comparison of vectors, [lexicographically](core::cmp::std#lexicographical-comparison).
```